suc/pkg/rancher/connectionInfo.go

```diff
@@ -27,5 +27,7 @@ func UpdateConnectionInformation() (string, error) {
 	if err != nil {
 		return string(o), fmt.Errorf("failed to update connection info: %w", err)
 	}
+
+	logrus.Info("Successfully updated connection info")
```

Comment:
When should a log message end with a period and when shouldn't it? I won't block on this, just food for thought.